PBM-1527 Release Notes for 2.9.1 (#267)
percona/pbm-docs: docs/release-notes/2.9.1.md

# {{pbm.full_name}} 2.9.1 ({{date.2_9_1}})

[Installation](../installation.md){.md-button}

{{pbm.full_name}} is a distributed, low-impact solution for creating consistent backups of MongoDB sharded clusters and replica sets, and for restoring those backups to a specific point in time.

> This release fixes physical backups and restores in mixed deployments that have both MongoDB Community and Percona Server for MongoDB. 

Reviewer: ```suggestion
This release fixes the issue that prevented PBM from taking physical backups in deployments using both MongoDB Community and Percona Server for MongoDB.
```

> This is a hot-fix release to version 2.9.0. If you're running a mixed deployment with MongoDB Community and Percona Server for MongoDB, this update may help you in running physical backup and restores. 

Reviewer 2: ```suggestion
This release fixes physical backups and restores in mixed deployments that have both MongoDB Community and Percona Server for MongoDB. 
```

## Changelog

### Bugs Fixed

> * [PBM-1512](https://perconadev.atlassian.net/browse/PBM-1512) - Fixed the issue in PBM unable to take physical backup in deployments with MongoDB Community and Percona Server for  MongoBD

Reviewer: ```suggestion
* [PBM-1512](https://perconadev.atlassian.net/browse/PBM-1512) - Resolved the issue preventing PBM from taking physical backups in deployments with MongoDB Community and Percona Server for MongoDB
```
